src/SFA.DAS.AssessorService.Settings/IWebConfiguration.cs

```diff
@@ -2,24 +2,19 @@
 {
     public interface IWebConfiguration
     {
-        AuthSettings Authentication { get; set; }
         ApiAuthentication ApiAuthentication { get; set; }
         AzureApiAuthentication AzureApiAuthentication { get; set; }
         ClientApiAuthentication ClientApiAuthentication { get; set; }
```

Comment:
There are several uses of the ClientApiAuthentication - the use of this one is for the Assessor Web to connect to the Assessor Api; so this one should be called AssessorApiAuthentication